apps/cowswap-frontend/src/modules/swap/services/swapFlow/index.ts

```diff
@@ -71,19 +64,8 @@ export async function swapFlow(
       generatePermitHook,
     })
 
-    if (appDataContainsPermitSigner(orderParams.appData.fullAppData)) {
+    if (callDataContainsPermitSigner(orderParams.appData.fullAppData)) {
       reportPermitWithDefaultSigner(orderParams)
```

Comment:
Not in the original code, but we probably should remove permit that uses the permit signer if it ever gets into this.